src/components/UriViewer.js

```diff
@@ -120,6 +121,7 @@ const DownloadButton = ({ uri, metadata: { bucket, name, fileName, size } }) =>
       div({ style: { display: 'flex', justifyContent: 'center' } }, [
         h(ButtonPrimary, {
           disabled: !url,
+          onClick: () => Ajax().Metrics.captureEvent(Events.download, { downloadFrom: 'bam file' }),
           href: url,
```

Comment:
@panentheos There's no weirdness with having both an `onClick` and an `href`, right? I don't remember any offhand.

Reply:
It occurs to me that this will fire on every download, do we want to scope it down a bit more? Maybe make it conditional on the extension of the file?

Reply:
Good catch.  I'll play around it with it a bit and see if I can restrict to .bam extensions.

Reply:
Maybe just ask Janki if we want an event for every download, and pull the extension dynamically?

Reply:
I will ask!